### DISCUSSION TITLE
Welcome to Zodiac_Project Discussions!

### DISCUSSION THREAD

**Discussion by Bhavesh-Kapur** [Bhavesh-Kapur/Zodiac_Project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bhavesh-Kapur** [comment]
hi any updates on the page 2 ??

**Reply by sidagl** [reply]
hi, yeah I am on it..

**Comment by Bhavesh-Kapur** [comment]
project's backend is done. Now we need to just keep improving the css

**Comment by sidagl** [comment]
Flip card done, now you can connect it to the backend

**Reply by Bhavesh-Kapur** [reply]
okay thanks for the html I have connected it to backend sync in to see the changes on the local machine as well
